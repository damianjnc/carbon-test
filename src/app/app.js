// @flow

import React from 'react'

import Chart from 'screens/Chart'
import { Wrapper } from './styled'

export function App() {
  return (
    <Wrapper>
      <Chart />
    </Wrapper>
  )
}

export default App
