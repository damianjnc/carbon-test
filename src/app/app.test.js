// @flow

import React from 'react'
import { shallow } from 'enzyme'

import { App } from './app'

jest.mock('screens/Chart', () => 'Chart')
jest.mock('./styled', () => ({
  Wrapper: 'Wrapper'
}))

describe('app', () => {
  it('renders', () => {
    const wrapper = shallow(<App />)

    expect(wrapper).toMatchSnapshot()
  })
})
