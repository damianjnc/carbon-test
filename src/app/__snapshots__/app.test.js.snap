// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`app renders 1`] = `
<Wrapper>
  <Chart />
</Wrapper>
`;
